src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -162,11 +162,9 @@ public function afterCreating($class, $callback)
      * @param  callable  $callback
      * @return $this
      */
-    public function afterCreatingState($class, $state, $callback)
+    public function afterCreatingState($class, $state, callable $callback)
```

Comment:
If you want to add this, please do it on master.

Reply:
I did and @taylorotwell said to put it here. https://github.com/laravel/framework/pull/23670

Reply:
The feature has not been tagged yet so no worries about breaking changes.

Reply:
Ah, right, good.